t/14-grutasks.t

```diff
@@ -852,5 +860,6 @@ done_testing();
 # break subsequent tests; can happen if a subtest creates a task but
 # does not execute it, or we crash partway through a subtest...
 END {
+    chdir $FindBin::Bin;
```

Comment:
Do we need this with the scope guard?

Reply:
I don't know. I had some problems in the failing case, but I removed it now.